source/DSCResources/MSFT_xDnsRecordMx/MSFT_xDnsRecordMx.psm1

```diff
@@ -0,0 +1,286 @@
+$script:dscResourceCommonPath = Join-Path -Path $PSScriptRoot -ChildPath '..\..\Modules\DscResource.Common'
+$script:dnsServerDscCommonPath = Join-Path -Path $PSScriptRoot -ChildPath '..\..\Modules\DnsServerDsc.Common'
+
+Import-Module -Name $script:dscResourceCommonPath
+Import-Module -Name $script:dnsServerDscCommonPath
+
+$script:localizedData = Get-LocalizedData -DefaultUICulture 'en-US'
+
+<#
+    .SYNOPSIS
+        This will return the current state of the resource.
+
+    .PARAMETER Name
+        Specifies the name of the DNS server resource record object. For records in the apex of the domain, use a period.
+
+    .PARAMETER Zone
+        Specifies the name of a DNS zone.
+
+    .PARAMETER Target
+        Specifies the Target Hostname or IP Address.
+
+    .PARAMETER Priority
+        Specifies the Priority value of the MX record.
+
+    .PARAMETER TTL
+        Specifies the TTL value of the MX record. Value must be in valid TimeSpan format.
+
+    .PARAMETER DnsServer
+        Name of the DnsServer to create the record on.
+
+    .PARAMETER Ensure
+        Whether the host record should be present or removed.
+#>
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Name,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Zone,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Target,
+
+        [Parameter()]
+        [System.UInt16]
+        $Priority,
+
+        [Parameter()]
+        [ValidateScript({$ts = New-TimeSpan; [system.timespan]::TryParse($_, [ref]$ts)})]
+        [System.String]
+        $TTL,
+
+        [Parameter()]
+        [System.String]
+        $DnsServer = "localhost",
+
+        [Parameter()]
+        [ValidateSet('Present','Absent')]
+        [System.String]
+        $Ensure = 'Present'
+    )
+
+    Write-Verbose -Message ($script:localizedData.GettingDnsRecordMessage -f $Name, 'MX', $Zone, $DnsServer)
+
+    $DNSParameters = @{
+        Name         = $Name
+        ZoneName     = $Zone
+        ComputerName = $DnsServer
+        RRType       = 'Mx'
+    }
+
+    $record = Get-DnsServerResourceRecord @DNSParameters -ErrorAction SilentlyContinue
+
+    if ($null -eq $record)
+    {
+        return @{
+            Name = $Name.HostName;
+            Zone = $Zone;
+            Target = $Target;
+            Priority = $Priority
+            TTL = $TTL;
+            DnsServer = $DnsServer;
+            Ensure = 'Absent';
```

Comment:
Hashtables should be aligned and don't need the semi colons. Same on line 95-102.
```powershell
Name      = $Name.HostName
Zone      = $Zone
Target    = $Target
Priority  = $Priority
TTL       = $TTL
DnsServer = $DnsServer
Ensure    = 'Absent'
```